python/lsst/cp/pipe/linearity.py

```diff
@@ -166,24 +182,48 @@ def run(self, inputPtc, camera=None, inputDims=None):
             ampName = amp.getName()
             if (len(inputPtc.expIdMask[ampName]) == 0):
                 self.log.warn(f"Mask not found for {ampName} in non-linearity fit. Using all points.")
-                mask = np.repeat(True, len(inputPtc.rawExpTimes[ampName]))
+                mask = np.repeat(True, len(inputPtc.expIdMask[ampName]))
             else:
                 mask = inputPtc.expIdMask[ampName]
 
-            timeVector = np.array(inputPtc.rawExpTimes[ampName])[mask]
-            meanVector = np.array(inputPtc.rawMeans[ampName])[mask]
-
+            inputOrdinate = np.array(inputPtc.rawExpTimes[ampName])[mask]
+            inputAbscissa = np.array(inputPtc.rawMeans[ampName])[mask]
```

Comment:
I think I got myself confused using the fancy name instead of keeping it in terms of expTime and flux.  I've corrected it, but will have to run through a few tests to confirm that I got it right.

Reply:
Testing has confirmed that I did get it wrong, then even more wrong in fixing it, and now it is correct, based on the outputs of the debug code (plots appended to Jira ticket).